### ISSUE TITLE
drodownHandlerRenderer, close on click when select open

### ISSUE THREAD

**Issue by marysomerville** [sanusart/react-dropdown-select#82]
Hi, I am using the drodownHandlerRenderer to render a custom select icon. 
My issue is I cannot get it to close when the select is open and I click on the icon again. 
Reading the docs I thought that the method dropdown("toggle") would be for this but I can't get it working. 
I have maybe implemented it wrong? 

I get the error `Uncaught TypeError: methods.dropdown is not a function` 

![dropdown-select](https://user-images.githubusercontent.com/5196339/77520386-0f005680-6e79-11ea-9018-ba1d97830a74.PNG)

```<StyledWrapper className={classNames}>
      <Dropdown
        placeholder={
          attributes.defaultValue
            ? trans(attributes.defaultValue["label"])
            : `${trans("Choose")}...`
        }
        className="dropdown"
        options={options}
        color="#333"
        dropdownHandleRenderer={({ props, state, methods }) => (
          <span
            className="icon-font icon-unfold-more"
            onClick={() => methods.dropdown("toggle")}
          ></span>
        )}
        dropdownGap={-4}
        onBlur={handleBlur}
        width={width}
        searchable={false}
        {...attributes}
        isOpen={isOpen ? true : false}
      />
      <StyledLabel touched={isTouched}>
        {trans(label)}
        {required && <StyledRequired>*</StyledRequired>}
      </StyledLabel>
    </StyledWrapper>```

Package version  "4.2.0"
Typescript

**Comment by sanusart** [comment]
@marysomerville I think it should be `methods.dropDown("toggle")` note the Upercase D in drop**D**own

**Comment by marysomerville** [comment]
@sanusart ah jeez so it is. I have changed it now. If i use toggle as the value the select seems to open and close immediately. Gif attached

![dropdown](https://user-images.githubusercontent.com/5196339/77748550-76a0d800-7018-11ea-8600-4eace7b23045.gif)

Am I missing any other settings?

**Comment by sanusart** [comment]
@marysomerville I might think there is a bug, 
I'll fix #61 later today and you'll be able to pass anything there.

**Comment by sanusart** [comment]
@marysomerville If you upgrade to 4.3.0 you can just skip the call for `methods.dropDown()` it will do that by itself. 
Example here: https://sanusart.github.io/react-dropdown-select/examples/#Custom-dropdown-handle

Cheers
